jams/core.py

```diff
@@ -496,49 +498,21 @@ class Sandbox(JObject):
     pass
 
 
-class JamsFrame(pd.DataFrame):
-    '''A data-frame class for JAMS.
+Observation = namedtuple('Observation',
+                         ['time', 'duration', 'value', 'confidence'])
+'''Core observation type: (time, duration, value, confidence).'''
 
-    This automates certain niceties, such as timestamp
-    conversion and serialization.
-    '''
-
-    __dense = False
-
-    def __init__(self, data=None, index=None, columns=None, dtype=None):
-        '''Construct a new JamsFrame object.
-
-        Parameters
-        ----------
-        data
-            Optional data for the new JamsFrame, in any format supported
-            by `pandas.DataFrame.__init__`.
 
-            Fields must be `['time', 'duration', 'value', 'confidence']`.
+class AnnotationData(object):
```

Comment:
fair enough ... I guess JamsFrame inherited from DataFrame, but maybe that was half the issue?

if we're not going to inherit from it, I'd propose making obs some level of private (dunders?) so that users are discouraged from using it as part of the API. However, I don't like having to wrap a bunch of pre-existing functionality ... SortedListWithKey is quite fully loaded.

Also worth mentioning here: What gets added to an AnnotationData object? `Observation`s or the fields of an observation? or, said differently, who is responsible for creating the Observation object, the user or the AnnotationData object? because, if it's the former, I think that's an argument for inheritance. 

related pop quiz: in Jams, who creates an Annotation when adding it to a JAMS object, the user or the owning object? (because we should follow that paradigm for consistency)